File: board_dsl/src/main/java/com/dooho/board/api/auth/SignInResponseDto.java
package com.dooho.board.api.auth;

import com.dooho.board.api.user.UserEntity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SignInResponseDto {
    private String token;
    private Integer tokenExprTime;
    private String refreshToken;
    private Integer refreshExprTime;
    private UserEntity user;
}

File: board_Kotlin/src/main/java/com/dooho/board/api/auth/SignInResponseDto.kt
package com.dooho.board.api.auth

import com.dooho.board.api.user.UserEntity

data class SignInResponseDto (
    val token : String? = null,
    val tokenExprTime : Int? = null,
    val refreshToken : String? = null,
    val refreshTokenExprTime : Int? = null,
    val user : UserEntity? = null
)